test/requirements.txt

```diff
@@ -0,0 +1,2 @@
+avocado-framework==59.0
+avocado-framework-plugin-varianter-yaml-to-mux==59.0
```

Comment:
You have to include `avocado-qemu` in the requirements list (https://pypi.python.org/pypi/avocado-qemu)